src/views/user/LoginView.vue

```diff
@@ -22,6 +22,9 @@
         :label="$gettext('Password')"
         icon="key"
       />
+      <div class="login-view-warning" v-translate>
+        <b> Beware of case-sensitive login and password</b>
```

Comment:
leading space ?

Reply:
Do not use `<b>`, prefer `<span class="has-text-weight-bold">`

It seems way more complex, yes :)